scripts/build-ios-simulator.sh

```diff
@@ -2,6 +2,8 @@
 
 APP_NAME="Woof"
```

Comment:
You can make the variable constant.
IIRC, the syntax is:

```bash
readonly APP_NAME="Woof"
```

Reply:
Fixed with https://github.com/ios-course/ironfoudation-team-project/commit/e65e3269a0ac9e61c73db68409846e8f4db93658, yes it's a syntax for constant in bash